src/modes/map/map_treasure.cpp

```diff
@@ -319,14 +319,14 @@ void TreasureSupervisor::_UpdateList()
             _selection_name.SetText(UTranslate("Drunes"));
             _selection_icon = GlobalManager->Media().GetDrunesIcon();
 
-            // Plural gettext for the 1st sentence.
             const std::string temp_string =
+                // Part 2 of "With the additional %u drune(s) found in this treasure added, ... "
```

Comment:
This comment is for the benefit of translators, not for the benefit of developers. Translators cannot search their po-files for "what it does" to find the second half of the string. I'm an experienced localizer, so please trust me on this - it's what I would need to translate this properly. So yes, if we change the other sentence as developers, we will need to pay attention.